### PR TITLE
HEEDLS-314 Assessment page improve voice over text

### PR DESCRIPTION
**Pull request by kwunkiuma**
Split the post learning view model assessment status field.

Update view model tests.

Add hidden text to post learning assessment and diagnostic assessment pages.

Ran and passed all unit tests, tested screen reader in Firefox & Chrome.

### CODE REVIEW

File: DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/PostLearningAssessmentViewModelTests.cs [hunk 1]
```diff
@@ -98,34 +98,85 @@ public void Post_learning_assessment_assessment_status_with_no_attempts_should_b
         }
 
         [Test]
-        public void Post_learning_assessment_attempts_information_with_attempts_can_be_passed()
+        public void Post_learning_assessment_assessment_status_with_no_attempts_should_have_no_score_information()
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/PostLearningAssessmentViewModelTests.cs:101]: I think it makes sense for this to be with the other `ScoreInformation` tests 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {
 <p class="nhsuk-u-secondary-text-color nhsuk-u-margin-bottom-0">
   @Model.TimeSpentSummary spent
-  <span class="visually-hidden">on this tutorial</span>
+  <span class="nhsuk-u-visually-hidden">on this tutorial</span>
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml:8]: How did this work if we were using the wrong class? Is `visually-hidden` our own class, i.e. one we've added css for? 🙂

**Review by FridaTveit — COMMENTED** [review]
A couple of comments, otherwise looks good 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml (hunk 2 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml:8]: I asked @SChandarana and he doesn't really remember where he got it from, the `visually-hidden` class does exist in our CSS and is being used in some Learning Portal pages but we're using `nhsuk-u-visually-hidden` on all Learning Menu pages so I changed it for consistency.

Comment (FridaTveit) [reply DigitalLearningSolutions.Web/Views/LearningMenu/Shared/_TutorialTimeSummary.cshtml:8]: Okay, fair enough 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**